pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -269,6 +270,8 @@ func (m *manager) Allocate(pod *v1.Pod, container *v1.Container) error {
 		klog.ErrorS(err, "Allocate error", "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
 		return err
 	}
+
+	klog.V(4).InfoS("Memory Allocated by memorymanager successfully", "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name)
```

Comment:
ditto

Reply:
Moved to static policy but just not sure if it makes sense to add `containerBlocks` to the logs to capture the allocated memory. I am leaning towards not...